src/Message/SmsMessage.php

```diff
@@ -17,6 +17,13 @@ class SmsMessage implements SmsMessageInterface {
   protected $uuid;
 
   /**
+   * The sender's name.
+   *
+   * @var string
```

Comment:
Will fix. This was just a cut and paste of existing code. As an aside, per Drupal code standards, null in docs is not capitalized. But seeing as that's what we have in many other places, we can do that in a separate issue.